AppController/djinn.rb

```diff
@@ -2966,7 +3105,10 @@ def restore_appcontroller_state()
 
     # Now that we've restored our state, update the pointer that indicates
     # which node in @nodes is ours
-    find_me_in_locations()
+    find_me_in_locations
```

Comment:
We are very inconsistent about this. When I first joined, the policy was to include the parentheses. However, the only style guides I was able to find seem to lean towards omitting them. I don't have a strong opinion, so I'm fine with omitting them unless someone disagrees.